src/commands/images/buildImage.ts

```diff
@@ -20,37 +22,31 @@ export async function buildImage(context: IActionContext, dockerFileUri: vscode.
 
     const dockerFileItem = await quickPickDockerFileItem(context, dockerFileUri, rootFolder);
 
-    let contextPath: string = dockerFileItem.relativeFolderPath;
-    if (defaultContextPath && defaultContextPath !== '') {
-        contextPath = defaultContextPath;
-    }
-    let absFilePath: string = path.join(rootFolder.uri.fsPath, dockerFileItem.relativeFilePath);
-    let dockerFileKey = `buildTag_${absFilePath}`;
-    let prevImageName: string | undefined = ext.context.globalState.get(dockerFileKey);
-    let suggestedImageName: string;
-
-    if (!prevImageName) {
-        // Get imageName based on name of subfolder containing the Dockerfile, or else workspacefolder
-        suggestedImageName = path.basename(dockerFileItem.relativeFolderPath).toLowerCase();
-        if (suggestedImageName === '.') {
-            suggestedImageName = path.basename(rootFolder.uri.fsPath).toLowerCase();
-        }
-
-        suggestedImageName += ":latest"
+    const task = await getOfficialBuildTaskForDockerfile(dockerFileItem.absoluteFilePath, rootFolder);
+    if (task) {
+        await vscode.tasks.executeTask(task);
```

Comment:
Just a side note, but the user's default terminal can definitely cause issues. Make sure to test with cmd, PowerShell, and git bash as your default terminal on Windows

Reply:
Will do.

Reply:
CWD and PowerShell work fine but Git Bash is replacing `/dev/null` in our run command with `nul`. Stupid shell. Need to figure out how to override it.

Reply:
Well, couldn't figure out how to override it, but I found that the entrypoint wasn't even needed. Tested all three across both Windows and Linux containers. Resolved in #1278.